app/controllers/api/services_controller.rb

```diff
@@ -84,6 +85,15 @@ def service_params
     params.require(:service).permit(permitted_params)
   end
 
+  # This will be the default 'settings' when apiap is live
+  def settings_apiap
+    render :settings_apiap
+  end
+
+  def apiap?
```

Comment:
Extract it somewhere else?
Making it as a helper to be used in the views cannot work if we do not merge this sadly https://github.com/3scale/porta/pull/1233/